collect_app/src/main/res/values-cs/strings.xml

```diff
@@ -438,7 +438,6 @@
   <string name="sort_by_status_asc">Stav, dokončen první</string>
   <string name="sort_by_status_desc">Stav, nedokončen první</string>
   <string name="sort_the_list">Řazení seznamu</string>
-  <string name="request_permissions_google_account">ODK Collect potřebuje přístup k vašemu účtu Google, aby získal prázdné formuláře a odeslal vyplněné formuláře.</string>
```

Comment:
What's the reason behind this change? It doesn't seem like it should be here!

Reply:
It doesn't! Not sure how it slipped into master but every time lint is run it fails due to it. I should have just sent it in as another PR so it could be merged in quickly.